### PR TITLE
[7] Develop a Contact Form Page using HTML

### PR DESCRIPTION
**Pull request by kazeemerinfolami**
## Task: `[7] Develop a Contact Form Page using HTML`
- `Contact Form`
Screen Shot 2020-09-20 at 9 13 54 PM

### CODE REVIEW

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    <form>
+      <div>
+        <h1>Contact Form</h1>
+      </div>
+      <hr />
+      <label for="name"><b>Full Name </b>:</label>
+      <input type="text" placeholder="name" id="name" name="name" /><br /><br />
+      <label for="email"><b>Email</b> :</label>
+      <input
+        type="email"
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/htmlForms/contact.html:18]: make the input tag and all its attributes take a single line

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        placeholder="email"
+        id="email"
+        name="email"
+      /><br /><br />
+      <label for="subject"><b>Subject </b>:</label>
+      <input
+        type="text"
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/htmlForms/contact.html:25]: same here

File: submissions/kazeemErinfolami/htmlForms/contact.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        placeholder="Subject"
+        id="subject"
+        name="subject"
+      /><br /><br />
+      <label for="message-input"><b>Text </b>:</label>
+      <textarea
+        placeholder="message input"
```
Comment (adaugochi) [inline submissions/kazeemErinfolami/htmlForms/contact.html:32]: and here

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/kazeemErinfolami/htmlForms/contact.html (hunk 3 above)
Comment (kazeemerinfolami) [reply submissions/kazeemErinfolami/htmlForms/contact.html]: Okay, I am using prettier to format my code.

**Pull request closed without merging**